src/lib/stores/marketplace.ts

```diff
@@ -0,0 +1,686 @@
+            },
+            {
+                name: 'APPWRITE_ENDPOINT',
+                description: `The URL endpoint of the Appwrite server. <a class="u-bold" target="_blank" href="https://appwrite.io/docs/getting-started-for-server">Learn More</a>`,
+                value: 'https://cloud.appwrite.io/v1',
+                placeholder: 'https://cloud.appwrite.io/v1',
+                required: false
+            }
+        ]
+    },
+    {
+        icon: 'icon-vonage',
+        id: 'whatsapp-with-vonage',
+        name: 'WhatsApp with Vonage',
+        tagline: 'Simple bot to answer WhatsApp messages.',
+        permissions: ['any'],
+        events: [],
+        cron: '',
+        timeout: 15,
+        runtimes: [
+            {
+                name: 'node-18.0',
+                commands: 'npm install',
+                entrypoint: 'src/main.js',
+                providerRootDirectory: 'node/whatsapp-with-vonage'
+            }
+        ],
+        instructions: `For documentation and instructions check out [README.md](https://github.com/appwrite/templates/tree/main/node/whatsapp-with-vonage) file.`,
+        vcsProvider: 'github',
+        providerRepositoryId: 'templates',
+        providerOwner: 'appwrite',
+        providerBranch: 'main',
+        variables: [
+            {
+                name: 'VONAGE_API_KEY',
+                description: `API Key to use the Vonage API. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/204014493-How-do-I-find-my-Voice-API-key-and-API-secret-">Learn More</a>`,
+                value: '',
+                placeholder: '62...97',
+                required: true
+            },
+            {
+                name: 'VONAGE_API_SECRET',
+                description: `Secret to use the Vonage API. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/204014493-How-do-I-find-my-Voice-API-key-and-API-secret-">Learn More</a>`,
+                placeholder: 'Zjc...5PH',
+                required: true
+            },
+            {
+                name: 'VONAGE_API_SIGNATURE_SECRET',
+                description: `Secret to verify the JWT token sent by Vonage. <a class="u-bold" target="_blank" href="https://developer.vonage.com/en/getting-started/concepts/signing-messages">Learn More</a>`,
+                placeholder: 'NXOi3...IBHDa',
+                required: true
+            },
+            {
+                name: 'VONAGE_WHATSAPP_NUMBER',
+                description: `Vonage WhatsApp number to send messages from. <a class="u-bold" target="_blank" href="https://api.support.vonage.com/hc/en-us/articles/4431993282580-Where-do-I-find-my-WhatsApp-Number-Certificate-">Learn More</a>`,
+                placeholder: '+14000000102',
+                required: true
+            }
+        ]
+    },
+    {
+        icon: 'icon-bell',
+        id: 'push-notification-with-fcm',
+        name: 'Push Notification with FCM',
```

Comment:
```suggestion
        name: 'Push notification with FCM',
```

@Meldiron All of these names should be sentence case... sorry!